tests/test_client.py

```diff
@@ -310,6 +319,26 @@ def test_fetch_from_archive_exception(self):
         with self.assertRaises(requests.exceptions.HTTPError):
             _ = client.fetch(CLIENT_SPIDERMAN_URL)
 
+    @httpretty.activate
+    def test_sanitize_for_archive(self):
```

Comment:
Not sure what you are trying to test here. I would check that calling to `satanize_for_archive` method (not the overriden version in MockedClient) returns the expected result.